compiler/control/OMROptions.cpp

```diff
@@ -1242,6 +1242,8 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"unleashStaticFieldFolding",        "O\tbypass the class white-list, and allow static final fields to be folded aggressively", RESET_OPTION_BIT(TR_RestrictStaticFieldFolding), "F"},
    {"unresolvedSymbolsAreNotColdAtCold", "R\tMark unresolved symbols as cold blocks at cold or lower", SET_OPTION_BIT(TR_UnresolvedAreNotColdAtCold), "F"},
    {"upgradeBootstrapAtWarm",           "R\tRecompile bootstrap AOT methods at warm instead of cold", SET_OPTION_BIT(TR_UpgradeBootstrapAtWarm), "F", NOT_IN_SUBSET},
+   {"useFlattenedFieldArrayRuntimeHelpers",        "M\tuse the runtime helpers for flattened fields for array elements", SET_OPTION_BIT(TR_UseFlattenedFieldArrayRuntimeHelpers), "F"},
```

Comment:
I think the thing that holds the value type value is the thing that is flattened - so a "flattened field" or a "flattened array element."  In this case, I think the name should be something like "useFlattenedArrayElementRuntimeHelpers" and for the description "use runtime helpers for flattened array elements"